doc/ext/convert-svg-to-pdf.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+"""
+    Converts SVG images to PDF using chrome in case the builder does not
+    support SVG images natively (e.g. LaTeX).
+
+"""
+import subprocess
+
+from sphinx.errors import ExtensionError
+from sphinx.locale import __
+from sphinx.transforms.post_transforms.images import ImageConverter
+from sphinx.util import logging
+from errno import ENOENT, EPIPE, EINVAL
+import os
+
+if False:
+    # For type annotation
+    from typing import Any, Dict  # NOQA
+    from sphinx.application import Sphinx  # NOQA
+
+
+logger = logging.getLogger(__name__)
+
+
+class Converter(ImageConverter):
+    conversion_rules = [
+        ('image/svg+xml', 'application/pdf'),
```

Comment:
Does this make it so this only runs when it needs a PDF (i.e., only for the latex build)?

Reply:
We just need to change conversion-rule to run it for any other image type,  further we can change its parent class ImageConverter to get our extention to run on different file types as well,  this extension will run whenever it sees conversion requirements of (image/svg+xml, application/pdf)